waku/v2/protocol/waku_swap/waku_swap_types.nim

```diff
@@ -6,6 +6,19 @@ import
   ../../node/peer_manager/peer_manager  
 
 type
+  # The Swap Mode determines the functionality available in the swap protocol.
```

Comment:
I'd make the docstring self contained and explain briefly what each mode implies here, as opposed to linking to a (possibly in the future outdated) discussion